run/interpreter.go

```diff
@@ -100,7 +100,9 @@ func (i interpreter) executeShell(
 	if err != nil {
 		return fmt.Errorf("failed to parse task: %w", err)
 	}
-	if os.Getenv("NO_COLOR") != "1" && term.IsTerminal(int(os.Stdout.Fd())) {
+	// Ignore G115: Potential integer overflow when converting between integer types
```

Comment:
I think the nosec line needs be precede the offending line, if I remember correctly!